packages/block-editor/src/store/selectors.js

```diff
@@ -2929,6 +2931,11 @@ export const getBlockEditingMode = createRegistrySelector(
 			if ( clientId === null ) {
 				clientId = '';
 			}
+
+			if ( getParentPatternCount( state, clientId ) > 0 ) {
+				return getPatternBlockEditingMode( state, clientId );
+			}
```

Comment:
I haven't tested this with 'zoom-out' mode yet, so it might need some adjustment.

Reply:
https://github.com/user-attachments/assets/0e8456ad-7271-43f2-b542-6bd0a6bc5b36

I tested it. The only quirk is that the overridable fields highlight when the block is clicked.

Reply:
The code for flashing blocks is part of the props passed to `useFlashEditableBlocks` hook in `packages/block-editor/src/components/block-list/use-block-props/index.js`

Reply:
It would actually be a good feature _if_ we could edit those fields in zoom out view. In the future we _might_ want to allow that.

Reply:
Ah, I see, I didn't realize they weren't editable. I've pushed a commit that adjusts that.